libs/pages/elewa/social-impact/src/lib/pages/social-impact-page/social-impact-page.component.html

```diff
@@ -1 +1 @@
-<p>social-impact-page works!</p>
+<elewa-group-elewa-group-image-and-text-banner></elewa-group-elewa-group-image-and-text-banner>
```

Comment:
This issue is not completed,

As lemmy described, create a component "elewa-social-impact-cooperative-section" add the text and image banner there.